### ISSUE TITLE
The <audioCodec> tag does not work when renaming files, dry run or commit.

### ISSUE THREAD

**Issue by oliviomoura** [Komet/MediaElch#1212]
Describe the bug
The <audioCodec> tag does not work when renaming files, dry run or commit.

To Reproduce

Start MediaElch.
select movie
load stream details
check for movie audio and video details exist for the movie
save movie
Select rename movie filename, place something like "<title>.{3D}3D.{/3D}<audioCodec>.<videoCodec>.<extension>"
Click Dry-Run
Expected behaviors:
1-the audio tag is not show, no audio tag at the filename
2-the movie to be rename is not show, only metadata rename files are showed

MediaElch Version:

MediaElch 2.8.6 - Coridian

Compiled with Qt version 5.14.2 (64 Release, GCC 5.5.0)
Using Qt version 5.14.2
System: Windows Server 2016 Version 1809 (x86_64-little_endian-llp64)

MediaInfo Version: MediaInfoLib - v20.03

**Comment by bugwelle** [comment]
Hi,

first of all: Thanks for providing the developer details. I think you're the first person to post it here. :-)

I don't get your two points:

> 1-the audio tag is not show, no audio tag at the filename
2-the movie to be rename is not show, only metadata rename files are showed

Could you explain them in more detail? :)

I tested MediaElch and it turns out that only know audio codec are used. I'll adapt MediaElch to also use unknown ones. With that, e.g. `aac` will also be displayed.

**Comment by oliviomouraoutsystems** [comment]
It's that, the acc is not displayed. The app should read what comes from mediainfo and drop it at the filename, no need to check for know/unknown codecs, should just trust what mediainfo returns.. just my point of view.

As for the rename do not show the video files please check this image:
https://pasteboard.co/JMOO9Ir.png

I have 7 files at the folder, but only 6 are show, the missing one is the video one ( mp4 )

**Comment by oliviomoura** [comment]
It's that, the acc is not displayed. The app should read what comes from mediainfo and drop it at the filename, no need to check for know/unknown codecs, should just trust what mediainfo returns.. just my point of view.

As for the rename do not show the video files please check this image:
https://pasteboard.co/JMOO9Ir.png

I have 7 files at the folder, but only 6 are show, the missing one is the video one ( mp4 )

**Comment by bugwelle** [comment]
> should just trust what mediainfo returns.. just my point of view.

I agree. The latest nightly has this fixed for `audioCodec`. I don't know why MediaElch looked for known codecs in the first place...

I'll look into the missing file issue tomorrow. :)
